src/k8s-engine/lima.ts

```diff
@@ -653,7 +653,7 @@ export default class LimaBackend extends events.EventEmitter implements K8s.Kube
     if (!dockerPortForwards) {
       config.portForwards?.push({
         guestSocket: '/var/run/docker.sock',
-        hostSocket:  'docker',
+        hostSocket:  path.join(paths.altAppHome, 'docker.sock'),
       });
     }
   }
```

Comment:
This code only adds a portforwarding rule for `/var/run/docker.sock` if no such rule exists yet. But it will not update the rule when it already exists.

We should first delete any rule that forwards `/var/run/docker.sock` → `docker` first, so we'll keep any modified rule, but replace the old default rule.

BTW, I'm not quite sure why we need to keep any non-default rule; I think that should not be supported, and we should unconditionally force the socket to be forwarded to `~/.rd/lima`. If the user needs a socket elsewhere, they can always add a symlink. But let me know if I missed the reason we do this!